crates/bevy_render/src/primitives/mod.rs

```diff
@@ -4,6 +4,18 @@ use bevy_reflect::Reflect;
 use bevy_utils::HashMap;
 
 /// An axis-aligned bounding box.
+///
+/// It represents a box covering the local space occupied by the entity, with faces
+/// orthogonal to the axis. It is used as a component on an entity to determine
+/// if it should be rendered by a [`Camera`](crate::camera::Camera) entity if it
+/// intersects with its [`Frustum`], in a process called frustum culling.
+///
+/// It will be added automatically to entities that could be subject to frustum
+/// culling, and don't have the [`NoFrustumCulling`](crate::view::visibility::NoFrustumCulling)
```

Comment:
It isn't but I'm not sure myself why would you want to opt out of it actually.

Reply:
See #4971

Reply:
This is because in animation the Mesh's vertices positions are updated, but the Aabb is not, right?

Lighting/reflection is for when you have a Mesh with a reflective material and you want to see something in it that is out of frustum, right?

I'm trying to write meaningful examples.